onnxruntime/python/tools/transformers/models/stable_diffusion/optimize_pipeline.py

```diff
@@ -46,9 +47,28 @@ def has_external_data(onnx_model_path):
     return False
 
 
+def is_sd_3(source_dir: Path):
+    return (source_dir / "text_encoder_3").exists()
+
+
+def is_sdxl(source_dir: Path):
+    return (source_dir / "text_encoder_2").exists() and not (source_dir / "text_encoder_3").exists()
+
+
+def _get_model_list(source_dir: Path):
```

Comment:
Greetings!
It seems that model_list_flux is missing. Could you confirm if this is intentional or if it's something that still needs to be added?

Reply:
Flux optimization is still on-going.